WebRoot/public/extuserprov.jsp

```diff
@@ -147,5 +151,10 @@
 		return !isError;
 	}
 </script>
+<% if (request.getAttribute("errorCode") != null) { %>
+	<script>
+		document.getElementById("ZLoginErrorPanel").style.display = "block";
```

Comment:
Moving this to L66 seems to be a simpler fix and avoids this client side javascript hack altogether, no?